requirements/generated/requirements-base.txt

```diff
@@ -0,0 +1,189 @@
+#
```

Comment:
Was this generated manually by accident as we don't use requirements/generated/requirements-base.txt?

Reply:
removed